docs/installation/viam-micro-server-dev.md

```diff
@@ -20,275 +20,262 @@ aliases:
+   ```
 
-Then, install `dfu-util`:
+   Then, install dependencies:
 
-```sh { class="command-line" data-prompt="$"}
-brew install dfu-util
-```
+   ```sh { class="command-line" data-prompt="$"}
+   brew install cmake dfu-util ninja
+   ```
 
-    {{% /tab %}}
-    {{% /tabs %}}
+   {{% /tab %}}
+   {{< /tabs >}}
 
-2. Install `Rust` and `cargo`
+1. If you do not yet have a Rust environment installed, install it with `rustup`.
 
    ```sh { class="command-line" data-prompt="$"}
    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
    ```
 
-   Once completed open a new tab on your terminal or run `. "$HOME/.cargo/env"`.
+   Once completed, start a new shell instance, or run `. "$HOME/.cargo/env"` in the current one.
 
    See the [Rust Installation guide](https://www.rust-lang.org/tools/install) for more information and other installation methods.
 
-3. Install `cargo-generate` with `cargo`
-
-   Run the following command to install `cargo-generate`:
+1. Install `espup` and ESP development utilities with `cargo`:
 
    ```sh { class="command-line" data-prompt="$"}
-   cargo install cargo-generate
+   cargo install cargo-espflash cargo-generate espflash espup ldproxy
    ```
 
-4. Install `espflash`
-
-   Run the following command to install `espflash`:
+1. Use `espup` to download and install the ESP Rust toolchain:
 
    ```sh { class="command-line" data-prompt="$"}
-   cargo install espflash
+   espup install -s -f /dev/null -v 1.76.0
    ```
 
-## Set up your development environment
+{{< alert title="Note" color="tip" >}}
+Prior versions of the above `espup` instructions created a file called `export-rs.sh` which needed to be sourced before proceeding.
+That requirement [no longer applies](https://github.com/esp-rs/esp-idf-hal/issues/319#issuecomment-1785168921) for `viam-micro-server` development.
+The above command redirects `espup`'s production of the setup script to `/dev/null` to avoid cluttering the home directory.
+If you would like to instead retain the setup script, replace `/dev/null` in the above command with the location where you would like the script to be written, or remove the `-f /dev/null` entirely and the file will be written to `$HOME/export-esp.sh` by default.
+{{< /alert >}}
 
-{{< tabs >}}
-{{% tab name="Canon CLI (recommended)" %}}
+## Creating a project or module
 
-[Canon](https://github.com/viamrobotics/canon) is a CLI utility for managing a Docker-based canonical environment.
+With the dev setup for `viam-micro-server`, you can:
 
-1. [Install Docker Engine](https://docs.docker.com/engine/install/). Canon requires a working installation of Docker Engine.
-2. If you are running Docker Engine on Linux, make sure that you go through the [post installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to run Docker automatically on startup.
+- Creating a project that combines `viam-micro-server` with one or more modules to produce a new firmware instance with expanded functionality.
```

Comment:
I realize this was also a mistake in my suggestion but please make sure you read through suggestions before applying them.

```suggestion
- Create a project that combines `viam-micro-server` with one or more modules to produce a new firmware instance with expanded functionality.
```

Reply:
sorry, normally do